# HG changeset patch
# User Rik <rik@octave.org>
# Date 1424473183 28800
#      Fri Feb 20 14:59:43 2015 -0800
# Node ID ab1185e08273cc30d6fa50b14a1301e85fada56a
# Parent  a0c9c85860c06e6893ae0a97057a9a181f9738ab
maint: Move annotation.m from plot/draw to plot/appearance directory.

* scripts/plot/appearance/annotation.m: Moved from plot/draw.

* scripts/plot/appearance/module.mk: Add annotation.m to build system.

* scripts/plot/draw/module.mk: Remove annotation.m from build system.

diff --git a/scripts/plot/draw/annotation.m b/scripts/plot/appearance/annotation.m
rename from scripts/plot/draw/annotation.m
rename to scripts/plot/appearance/annotation.m
diff --git a/scripts/plot/appearance/module.mk b/scripts/plot/appearance/module.mk
--- a/scripts/plot/appearance/module.mk
+++ b/scripts/plot/appearance/module.mk
@@ -2,16 +2,17 @@ FCN_FILE_DIRS += plot/appearance
 
 plot_appearance_PRIVATE_FCN_FILES = \
   plot/appearance/private/__axis_limits__.m \
   plot/appearance/private/__axis_label__.m
 
 plot_appearance_FCN_FILES = \
   plot/appearance/__clabel__.m \
   plot/appearance/__getlegenddata__.m \
+  plot/appearance/annotation.m \
   plot/appearance/axis.m \
   plot/appearance/box.m \
   plot/appearance/caxis.m \
   plot/appearance/clabel.m \
   plot/appearance/daspect.m \
   plot/appearance/datetick.m \
   plot/appearance/diffuse.m \
   plot/appearance/grid.m \
diff --git a/scripts/plot/draw/module.mk b/scripts/plot/draw/module.mk
--- a/scripts/plot/draw/module.mk
+++ b/scripts/plot/draw/module.mk
@@ -13,17 +13,16 @@ plot_draw_PRIVATE_FCN_FILES = \
   plot/draw/private/__patch__.m \
   plot/draw/private/__pie__.m \
   plot/draw/private/__plt__.m \
   plot/draw/private/__quiver__.m \
   plot/draw/private/__scatter__.m \
   plot/draw/private/__stem__.m
 
 plot_draw_FCN_FILES = \
-  plot/draw/annotation.m \
   plot/draw/area.m \
   plot/draw/barh.m \
   plot/draw/bar.m \
   plot/draw/colorbar.m \
   plot/draw/comet3.m \
   plot/draw/comet.m \
   plot/draw/compass.m \
   plot/draw/contour3.m \
